app/views/admin/api_docs/base/index.html.slim

```diff
@@ -13,19 +13,23 @@ h2 ActiveDocs: Service Specs
       th Name
       th System Name
       th State
+      - if @service.blank?
+        th API
       th Swagger Version
       th class='operations' = link_to('Create a new spec', new_api_docs_service_path(@service), class: 'action add')
     tbody
-      - for service in @api_docs_services
+      - for api_docs in @api_docs_services
```

Comment:
That would be nice to have as a partial so it could benefit from caching

Reply:
Ouch, I forgot about this. Ok, I will tomorrow :+1: